packages/synthetix-main/contracts/interfaces/IIssueUSDModule.sol

```diff
@@ -0,0 +1,62 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../utils/CurvesLibrary.sol";
+
+/// @title Liquidates the collateral for an account in a pool
+interface IIssueUSDModule {
```

Comment:
Unsure if this is right name for this? Maybe `UsdIssuer`, but it also burns...

Reply:
it always burns.

im sure @ajsantander will come up with something :laughing: